docs/cli/cli-data.md

```diff
@@ -326,25 +324,212 @@ that are of standard (aka not test) quality. Therefore, these filters can be eas
-00:00 - order my asset - state: active
-{'_links': {'_self': 'https://api.planet.com/data/v1/assets/eyJpIjogIjIwMjIxMDAzXzAwMjcwNV8zOF8yNDYxIiwgImMiOiAiUFNTY2VuZSIsICJ0IjogImJhc2ljX3VkbTIiLCAiY3QiOiAiaXRlbS10eXBlIn0', 'activate': 'https://api.planet.com/data/v1/assets/eyJpIjogIjIwMjIxMDAzXzAwMjcwNV8zOF8yNDYxIiwgImMiOiAiUFNTY2VuZSIsICJ0IjogImJhc2ljX3VkbTIiLCAiY3QiOiAiaXRlbS10eXBlIn0/activate', 'type': 'https://api.planet.com/data/v1/asset-types/basic_udm2'}, '_permissions': ['download'], 'expires_at': '2023-03-02T19:30:48.942718', 'location': 'https://api.planet.com/data/v1/download?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJQYWtGNHZuUEs3WXRmSFNGUklHY2I3YTNXT3piaTlaam4zWUpZMmxnd0x5cVlFMVBRSHU5QXNCcjR5Q3FxSjBNbl9yN3VwVEFQYUI1ZzhYNUJmcDhmUT09IiwiZXhwIjoxNjc3Nzg1NDQ4LCJ0b2tlbl90eXBlIjoidHlwZWQtaXRlbSIsIml0ZW1fdHlwZV9pZCI6IlBTU2NlbmUiLCJpdGVtX2lkIjoiMjAyMjEwMDNfMDAyNzA1XzM4XzI0NjEiLCJhc3NldF90eXBlIjoiYmFzaWNfdWRtMiJ9.Dd0opDjW3bBS6qLLZoNiJkfBsO2n5Xz9pM5apEUz_K6viDPFexhJiy6bMbaySbby8W0YvuATdb1uYXS2FkweDg', 'md5_digest': '3a9f7dd1ce500f699d0a96afdd0e3aa2', 'status': 'active', 'type': 'basic_udm2'}
-/path/to/data/20221003_002705_38_2461_1A_udm2.tif: 100%|██████████████████████████████████| 3.16k/3.16k [00:00<00:00, 32.0MB/s]
+planet data filter --string-in strip_id 5743640 | planet data stats PSScene day -
 ```
 
-## Stats
+Or the number of PlanetScope scenes collected in California each year:
+
+```
+curl -s https://raw.githubusercontent.com/ropensci/geojsonio/main/inst/examples/california.geojson | \
+planet data filter --geom - | planet data stats PSScene year - | jq
+```
+
+Will result in output like:
+
+```json
+{
+  "buckets": [
+    {
+      "count": 5261,
+      "start_time": "2014-01-01T00:00:00.000000Z"
+    },
+    {
+      "count": 34377,
+      "start_time": "2015-01-01T00:00:00.000000Z"
+    },
+    {
+      "count": 112331,
+      "start_time": "2016-01-01T00:00:00.000000Z"
+    },
+    {
+      "count": 504377,
+      "start_time": "2017-01-01T00:00:00.000000Z"
+    },
+    {
+      "count": 807086,
+      "start_time": "2018-01-01T00:00:00.000000Z"
+    },
+    {
+      "count": 806945,
+      "start_time": "2019-01-01T00:00:00.000000Z"
+    },
+    {
+      "count": 776757,
+      "start_time": "2020-01-01T00:00:00.000000Z"
+    },
+    {
+      "count": 684095,
+      "start_time": "2021-01-01T00:00:00.000000Z"
+    },
+    {
+      "count": 323557,
+      "start_time": "2022-01-01T00:00:00.000000Z"
+    },
+    {
+      "count": 56733,
+      "start_time": "2023-01-01T00:00:00.000000Z"
+    }
+  ],
+  "interval": "year",
+  "utc_offset": "+0h"
+}
+```
+
+You can see how the yearly output of Planet has gone up, though it actually went down in 2022 as the upgrade to 
+SuperDove meant much larger swaths, so the number of individual items went down even as we captured the whole
+earth.
+
+Note that the API does not support an 'all time' interval to get the total of all collections for an area, but
```

Comment:
There are 12 sentences that begin with "Note..." Remove some if they're not really notifications that need to be emphasized. For example, suggest changing this to:

```suggestion
The API does not support an 'all time' interval to get the total of all collections for an area, but
```